README.md

```diff
@@ -1,27 +1,98 @@
-# INSTALLATION
+# Installation
 
-### for python 2 installation ###
-pip install -r requirements.txt
+## Python 2
```

Comment:
is there any need to support Python2?

Reply:
I'm not sure. The project supported Python 2 beforehand so I didn't remove it. Besides, at least in my testing, sio2jail doesn't seem work on Python 3 right now (see #11), so I assumed there must be some Python 2 based deployments somewhere.

If there is no need to keep Python 2 working, then I can remove it.